### PR TITLE
[dotnet] Add future-proofing note on `Base64UrlEncoder`

### PR DESCRIPTION
**Pull request by RenderMichael**
### **User description**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

Addresses request in #14813

### Motivation and Context

This note will help us remember about the `Base64Url` type that is available on .NET 9+

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

___

### **PR Type**
documentation

___

### **Description**
- Enabled nullable reference types in the `Base64UrlEncoder` class to improve code safety.
- Added a comment to note the potential use of the built-in `Base64Url` type available in .NET 9+, addressing issue #14813.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>Base64UrlEncoder.cs</strong><dd><code>Add future-proofing note and enable nullable reference types</code></dd></summary>
<hr>

dotnet/src/webdriver/Internal/Base64UrlEncoder.cs

<li>Enabled nullable reference types.<br> <li> Added a comment about future usage of <code>Base64Url</code> type in .NET 9+.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/selenium/pull/14821/files#diff-35d9ef4edcc03326e175080418cb703fc44a741765ab59a58ed2039813178fef">+5/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

**Comment by RenderMichael** [comment]
Should this type be internal?

File: dotnet/src/webdriver/Internal/Base64UrlEncoder.cs [hunk 1]
```diff
@@ -19,8 +19,13 @@
 
 using System;
 
+#nullable enable
+
 namespace OpenQA.Selenium.Internal
 {
+    // When we target .NET 9+, we can use the built-in Base64Url type
```
Comment (nvborisenko) [inline dotnet/src/webdriver/Internal/Base64UrlEncoder.cs]: Somewhere in this file there is already a link to a source where we stole it.

So on top of this file I would expect:
```
/*
Based on: https:// github...(link in this file)
Now it is a part of .NET 9 (System.Blah.EncodedUrl type)
/*
```

Comment (RenderMichael) [reply dotnet/src/webdriver/Internal/Base64UrlEncoder.cs]: What do you think of the new message?

Comment (nvborisenko) [reply dotnet/src/webdriver/Internal/Base64UrlEncoder.cs]: I am OK (in this case) to have a ref from source code to BTS. Please remove a link somewhere in the middle of file.

Comment (RenderMichael) [reply dotnet/src/webdriver/Internal/Base64UrlEncoder.cs]: done

**Comment by nvborisenko** [comment]
We kept origin, goal, context. Thank you Michael.

**Pull request merged**